core/src/main/java/org/mobilitydata/gtfsvalidator/input/GtfsInput.java

```diff
@@ -44,6 +44,8 @@ public abstract class GtfsInput implements Closeable {
   public static final String invalidInputMessage =
       "At least 1 GTFS file is in a subfolder. All GTFS files must reside at the root level directly.";
 
+  public static boolean isMacZip = false;
```

Comment:
This variable should not be static. 
If we call hasSubfolderWithGtfsFile twice, once with a mac zip file and the second time with a non-mac zip file, isMacZip will be set to true for the first file, and will keep being true for the second file. But it's not a mac zip file!
Can't you determine if it's a mac zip file directly in the GtfsZipFileInput constructor instead of doing it in  containsGtfsFileInSubfolder? That way the isMacZip variable will be confined in the GtfsZipFileInput class.

Reply:
This is a very good point. I've made the changes.